src/routes/auth_routes.js

```diff
@@ -0,0 +1,7 @@
+module.exports = (app) => {
+  app.route('/auth').get((req, res) => {
```

Comment:
kiểu này thì ko add namespace vào đc, phải add thủ công từng api 1, dùng router nhé
https://scotch.io/tutorials/learn-to-use-the-new-router-in-expressjs-4